src/tests/test_connection.py

```diff
@@ -1184,3 +1185,55 @@ def test_database_in_directory_that_doesnt_exist(tmp_empty, uri, expected):
         SQLAlchemyConnection(engine=create_engine(uri))
 
     assert expected in str(excinfo.value)
+
+
+query_expected_outputs = [
```

Comment:
there's no need to declare a separate variable. you can just include this in the parameterize decorator

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/895/files#diff-ecfdf2a1f3adc925d3c32b8aa52540474a10a4160d552f281d7dbd85a0260170R1190)